docs/Cloud_integrations/Device_Integration_Center/kafka.md

```diff
@@ -86,7 +82,7 @@ sasl.mechanism=PLAIN
 === "Confluent’s Kafka"
     When using **Confluent’s Kafka**, copy the connection settings from your topic configuration view. We recommend copying only the basic configuration settings mentioned above. (See highlighted part in the visual below.)
```

Comment:
Suggestion:

When using **Confluent’s Kafka**, copy the connection settings from your topic configuration view. 
!!! info
We recommend copying only the basic configuration settings mentioned above. See highlighted part in the visual below.